source/includes/collections/behavior-capped-collections.rst

```diff
@@ -0,0 +1,19 @@
+Starting in 1.3.0, {+c2c-product-name+} supports :ref:`capped
+collections <manual-capped-collection>` with some limitations.
+
+- The minimum server version is 6.0.
+- :dbcommand:`convertToCapped` is not supported. If you run
+  ``convertToCapped``, ``mongosync`` exists with an error.
+- :dbcommand:`cloneCollectionAsCapped` is not supported. If you run
+  ``cloneCollectionAsCapped``, the ``mongosync`` behavior is undefined.
```

Comment:
I'm not sure I understand what we mean by 'undefined behavior'.  If it's just that we don't know what will happen if the user does this, it may be better to drop the second sentence altogether.